### PR TITLE
Upgrade flyway to 851

### LINKED ISSUE #5059: [SNYK: High] Remote Code Execution (RCE) (Due 03/18/2022)
**Overview**
[org.postgresql:postgresql](https://mvnrepository.com/artifact/org.postgresql/postgresql) is a Java JDBC 4.2 (JRE 8+) driver for PostgreSQL database. 

Affected versions of this package are vulnerable to Remote Code Execution (RCE) when using certain plugin features. pgjdbc instantiates plugin instances based on class names provided via authenticationPluginClassName, sslhostnameverifier, socketFactory, sslfactory, and sslpasswordcallback connection properties.

However, the driver did not verify if the class implements the expected interface before instantiating the class.

[SNYK-JAVA-ORGPOSTGRESQL-2390459](https://app.snyk.io/vuln/SNYK-JAVA-ORGPOSTGRESQL-2390459)

**Considerations:**
- This is tied to flyway. Perhaps we need to update flyway to 8.5.1. After upgrading check to see if postgresql driver is updated to 42.2.25 or 42.3.2 or higher.

**How to fix?**
- Upgrade org.postgresql:postgresql to version 42.2.25, 42.3.2 or higher.

### PR DESCRIPTION
**Pull request by fec-jli**
## Summary (required)

- Resolves #5059 

This PR upgrade flyway to version 8.5.1

### Required reviewers

1-2 developer

## How to test
1. Test locally, Ref: [https://github.com/fecgov/openFEC#installing-flyway](https://github.com/fecgov/openFEC#installing-flyway)
For MacOS user, open Safari browser to download
flyway-commandline-8.5.1-macosx-x64.tar.gz 
or you can download from google drive: [https://drive.google.com/drive/folders/1h5i4yL2pqlZc7ydSGahr_5ZZnVeUIGUJ](https://drive.google.com/drive/folders/1h5i4yL2pqlZc7ydSGahr_5ZZnVeUIGUJ)
update flyway conf file, activate virtue, run `invoke create_sample_db`
2. checkout feature branch
3. deploy to dev space without any error
4. Here is circleci log:
https://app.circleci.com/pipelines/github/fecgov/openFEC/1267/workflows/bb13e20e-79dc-4512-af90-2c4c6b1e5a8e/jobs/3768

Screen Shot 2022-03-04 at 1 21 26 PM

### CODE REVIEW

**Review by hcaofec — APPROVED** [review]
This looks great!  Thanks for updating readme.

File: .circleci/config.yml [hunk 1]
```diff
@@ -39,8 +39,8 @@ jobs:
           command: |
             sudo apt-get --allow-releaseinfo-change-suite update && sudo apt-get install -y default-jdk
             mkdir flyway
-            curl -s https://repo.maven.apache.org/maven2/org/flywaydb/flyway-commandline/8.4.3/flyway-commandline-8.4.3-linux-x64.tar.gz | tar zxv -C flyway
-            echo 'export PATH=./flyway/flyway-8.4.3:$PATH' >> $BASH_ENV
+            curl -s https://repo.maven.apache.org/maven2/org/flywaydb/flyway-commandline/8.5.1/flyway-commandline-8.5.1-linux-x64.tar.gz | tar zxv -C flyway
```
Comment (pkfec) [inline .circleci/config.yml:42]: @fec-jli I see flyway v8.5.2 is available. Any good reason you didn't  update flyway to the latest version?

**Comment by pkfec** [comment]
Remote Code Execution vulnerability exists in **flyway version 8.4.3.**  
Screen Shot 2022-03-07 at 5 52 22 PM


Remote Code Execution vulnerability is _FIXED_ in **flyway v8.5.1 or v8.5.2**
Screen Shot 2022-03-07 at 5 53 44 PM

**Comment by cnlucas** [comment]
This works for me, but I can re-review if you're switching to 8.5.2. Thanks!

**Pull request closed without merging**